src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -236,7 +245,7 @@ def prepare_logs_argument(c):
                 'main_entry', options_list=[
                     '--main-entry', '-m'], help="A string containing the path to the .NET executable relative to zip root.")
             c.argument(
-                'builder', help="The name of builder.", default="default")
+                'builder', help="The name of builder.", default="default", validator=validate_builder)
```

Comment:
add preview = true and note this is (enterprise only)

Reply:
OK, added.